modules/web-console/frontend/app/components/stacktrace-viewer-dialog/components/stacktrace-item/template.pug

```diff
@@ -0,0 +1,33 @@
+//-
+    Copyright 2019 GridGain Systems, Inc. and Contributors.
+
+    Licensed under the GridGain Community Edition License (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+.stacktrace-viewer__cause(
+    ng-click='$ctrl.toggleProperty()'
+)
+    .stacktrace-viewer__cause_message
+        .stacktrace-viewer__expander
+            div(ng-switch='$ctrl.expanded')
+                svg(ng-switch-when='true' ignite-icon='collapse')
+                svg(ng-switch-when='false' ignite-icon='expand')
+        span(ng-class='{bold: $ctrl.cause && $ctrl.expanded}') {{ $ctrl.cause.message }}
+.stacktrace-viewer__trace(
+    ng-show='$ctrl.expanded && $ctrl.cause.stacktrace'
+    ng-repeat='trace in $ctrl.cause.stacktrace track by $index'
+)
+    .stacktrace-viewer__trace_message
+        span {{ trace }}
+.stacktrace-viewer__trace(ng-if='$ctrl.expanded && !$ctrl.cause.stacktrace')
+    .stacktrace-viewer__trace_message
+        span Full stacktrace is available only in #[a(href="https://www.gridgain.com/resources/download" target="_blank") GridGain since 8.8.1] version
```

Comment:
I think this is not correct message "Full stacktrace is available only in GridGain since 8.8.1 version".
Lets show "Full stacktrace is not available"

Reply:
Message has been changed.